src/lib/features/change-request-segment-usage-service/sql-change-request-segment-usage-read-model.ts

```diff
@@ -1,5 +1,8 @@
 import { Db } from '../../db/db';
-import { IChangeRequestSegmentUsageReadModel } from './change-request-segment-usage-read-model';
+import {
+    ChangeRequestStrategy,
+    IChangeRequestSegmentUsageReadModel,
+} from './change-request-segment-usage-read-model';
 
 export class ChangeRequestSegmentUsageReadModel
```

Comment:
I would name the file as `change-request-segment-usage-read-model`

Reply:
Good input, and I would usually do the same. But I'm following the precedent set by the change request access read model (`src/lib/features/change-request-access-service/sql-change-request-access-read-model.ts`), so I think I'd like to keep it that way for now.